spec/game_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './game'
+
+describe Game do
+  subject { described_class.new(Date.new(2000, 1, 1), false, true, Date.new(2018, 1, 1)) }
+  describe '#move_to_archive' do
+    it 'should archive the game' do
+      subject.move_to_archive
+      expect(subject.archived).to eq(true)
+    end
+    it 'should not archive the game if it is not old enough' do
+      subject.last_played_at = Date.today
+      subject.move_to_archive
+      expect(subject.archived).to eq(false)
+    end
+  end
+end
```

Comment:
- The test code you provided looks good. It tests the behavior of the **move_to_archive** method of the Game class in two scenarios - when the game is old enough to be archived and when it is not. The test cases are clear and concise, and they use the **expect** syntax to make assertions about the behavior of the code being ### tested.
### 

### Good job @ZewdieMc 🥇